packages/edit-site/src/components/sidebar/template-card/index.js

```diff
@@ -36,9 +37,11 @@ export default function TemplateCard() {
 			<Icon className="edit-site-template-card__icon" icon={ icon } />
 			<div className="edit-site-template-card__content">
 				<h2 className="edit-site-template-card__title">{ title }</h2>
-				<span className="edit-site-template-card__description">
+				<div className="edit-site-template-card__description">
```

Comment:
Any reason why this wasn't a `<p>`?

Reply:
I'd expect it to be a `<p>` given `description` can't contain HTML.

Reply:
The description in all the blocks are all using [`<span>`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-card/index.js#L24) for some reasons. That's why I'm wondering if there are something we haven't thought of?